core/src/main/java/nl/nn/adapterframework/doc/BecomesMandatory.java

```diff
@@ -0,0 +1,25 @@
+/*
+   Copyright 2022 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.doc;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+public @interface BecomesMandatory {
```

Comment:
Ik heb het besproken met Niels, en wij bedachten dat het duidelijker was om niet een nieuwe apart Annotatie te maken, maar de bestaande Mandatory uit te breiden met een argument ignoreInCompatibilityMode, als in:
`@Mandatory(ignoreInCompatibilityMode=true)`

Reply:
Gedaan, ook in frank-doc project